CONTRIBUTING.md

```diff
@@ -32,6 +32,11 @@ Make the development version available globally:
 $ pip install -e .
 ```
 
+Install the `black` formatter hook with:
```

Comment:
Can you maybe add something a bit more explicit here like, we use `black -S` to format our repo. More info can be found ... and put in a link to the `black` docs

Reply:
do we want to mention about command line usage `black -S` or enforce using pre-commit hook? i will be giving a link to black docs.

Reply:
I think talking about command line usage for those that are interested / want to know, but then enforcing using pre-commit makes sense. Could just say this pre-commit equivalent to running `black -S`

Reply:
I can't tell that pre-commit equivalent to `black -S` as it is not true. We have our line length and so. But i think enforcing pre-commit and giving a link to black repo could be satisfactory. Those interested can browse the link and find out more. How does this sound to you? @sofroniewn

Reply:
Ok - no need to say equivalent, but can you at least say that we have `skip-string-normalization = true` and `max-line-length = 79`

Reply:
i think it will just be easier for people to see this information here rather than have to dig around the repo

Reply:
Done @sofroniewn please check